content/dart/concept/list/terms/remove/remove.md

```diff
@@ -0,0 +1,56 @@
+
+
+---
+Title: 'List.remove()'
+Description: 'Removes first occurrence of a value from the list.'
+Subjects:
+  - 'Andriod Development'
+  - 'Dart'
+Tags:
+  - 'List'
+  - 'Remove'
+  - 'Method'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+The **List.remove()** is an abstract method to remove the first occurence of an item from the list and return bool `true` if the item is present and is successfully removed, otherwise return bool `false` if the item is not present. The list must be growable list.
```

Comment:
```suggestion
The **`.remove()`** is a method to remove the first occurrence of an item from the list and return bool `true` if the item is present and is successfully removed, otherwise return bool `false` if the item is not present.
```
Add this line as a note after the introductory paragraph - 
`The list must be a growable list.`